docs/ref/layout.md

```diff
@@ -50,46 +50,46 @@ Only present when {question}`workflows` == `enhanced`.
+Handles workflows for Pull Requests and pushes to the `main` branch. Depending on {question}`workflows`, it either calls centralized workflows in [salt-extensions/central-artifacts](https://github.com/salt-extensions/central-artifacts/tree/main/.github/workflows) or local workflows in {path}`ci.yml <.github/workflows/ci.yml>`.
 
 :::{path} .github/workflows/tag.yml
 :::
 #### `.github/workflows/tag.yml`
-Workflow triggered by [tag pushes](publishing-target) for tags beginning with `v`. Depending on {question}`workflows`, either calls the centralized workflows in [salt-extensions/central-artifacts](https://github.com/salt-extensions/central-artifacts/tree/main/.github/workflows) or the local equivalent at {path}`.github/workflows/ci.yml`.
+Triggered by [tag pushes](publishing-target) for tags beginning with `v`. Similar to {path}`pr.yml <.github/workflows/pr.yml>`, it either calls centralized workflows or local ones in {path}`ci.yml <.github/workflows/ci.yml>`.
 
 :::{path} changelog
 :::
 ## `changelog`
-Directory containing [news fragments](news-fragment-target) for `towncrier`. Also contains the default version-specific changelog template at `changelog/.template.jinja`.
+Directory containing [news fragments](news-fragment-target) for `towncrier`. Also includes the default version-specific changelog template in `changelog/.template.jinja`.
 
 :::{path} docs
 :::
 ## `docs`
-Root directory for all documentation-related files.
+Root directory for documentation-related files.
 
 :::{path} docs/conf.py
 :::
 ### `docs/conf.py`
-Sphinx configuration and plugins.
+Contains Sphinx configuration and plugins.
 
 :::{path} docs/index.rst
 :::
 ### `docs/index.rst`
-The homepage of the documentation. Should (indirectly) link to all documentation files.
+Homepage for the documentation, (indirectly) linking to all other documentation files.
 
-:::{tip}
-If your Salt extension project has a `utils` directory, you need to include the corresponding documentation manually (the Copier template/pre-commit hook does not take care of that currently).
+:::{hint}
+If your project includes a `utils` directory, manually add the corresponding documentation here (not handled by the Copier template or the pre-commit hook).
 :::
 
 :::{path} docs/ref
 :::
 ### `docs/ref`
-Directory containing autogenerated documentation of your modules. Usually, you don't need to touch this directory, but it could be used e.g. for a configuration reference document.
+Directory containing autogenerated module documentation. Typically does not require manual updates, but can be used for custom documents, like a configuration reference.
 
 :::{path} docs/topics
 :::
 ### `docs/topics`
-Intended to contain high-level guides regarding your Salt extension, such as `Configuration`.
+Intended to hold high-level guides related to your Salt extension, such as `Configuration`. By default, includes a guide on `Installation`.
```

Comment:
By default, includes a guide on `Installation` -> By default, includes an `Installation` guide

IMO, this reads better